docs/code_patterns_and_conventions.md

```diff
@@ -40,8 +40,30 @@ When creating new JavaScript assets, use esbuild in `app/javascripts` rather tha
 When authorizing an action on an individual record, for example a Client, use a resource policy.
 
 ```ruby
-policy = user.policies.for_client(client)
-not_authorized! unless policy.show?
+policy = user.policy_for(client, type: client)
+not_authorized! unless policy.can_view?
```

Comment:
Design question, which seems better?
```ruby
policy.can_view?
# or
policy.view?
```

Other policy patterns I've used favor the latter, which is more succinct. However it is a further departure from our permission naming convention. Perhaps that's desirable though since the policy is no longer 1-1 with role; breaking the naming convention makes the lack of parity obvious.

Reply:
No strong preference from me.  I like how `prolicy.can_view?` reads, but I think we could all make the leap as well.